sk0-front/.prettierrc.toml

```diff
@@ -0,0 +1,7 @@
+trailingComma = "none"
+tabWidth = 2
+semi = true
+singleQuote = false
+arrowParens = "always"
+bracketSpacing = true
+endOfLine = "lf"
```

Comment:
should we set printWidth?

Reply:
I am happy with 80, but lets ask around.